webapp/src/index.tsx

```diff
@@ -317,6 +334,9 @@ export default class Plugin {
             Client4.setUrl(window.basename);
         }
 
+        const theme = getTheme(store.getState());
+        setCallsGlobalCSSVars(theme.sidebarBg);
```

Comment:
Another small bug I found when testing on Desktop.

Reply:
ah, that's why it was up above. What was the bug?